wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+
+namespace Intel.RealSense
+{
+    /// <summary>
+    /// Interaction logic for Window.xaml
+    /// </summary>
+    public partial class CaptureWindow : Window
+    {
+        private Pipeline pipeline;
+        private Colorizer colorizer;
+        private CancellationTokenSource tokenSource = new CancellationTokenSource();
+
+        static Action<VideoFrame> UpdateImage(Image img)
+        {
+            var wbmp = img.Source as WriteableBitmap;
+            return new Action<VideoFrame>(frame =>
+            {
+                if (frame == null) return;
+                
+                var rect = new Int32Rect(0, 0, frame.Width, frame.Height);
+                wbmp.WritePixels(rect, frame.Data, frame.Stride * frame.Height, frame.Stride);
+            });
+        }
+
+        public CaptureWindow()
+        {
+            InitializeComponent();
+
+            try
+            {
+                Action<VideoFrame> updateDepth;
+                Action<VideoFrame> updateColor;
+
+                // The colorizer processing block will be used to visualize the depth frames.
+                colorizer = new Colorizer();
+
+                // Create and config the pipeline to strem color and depth frames.
+                pipeline = new Pipeline();
+                var cfg = new Config();
+
+                using (var ctx = new Context())
+                {
+                    var devices = ctx.QueryDevices();
+                    if (0==devices.Count)

```

Comment:
formatting in the if statement (add spaces)

Reply:
done